### PR TITLE
Testing Math-Magicians

### PR DESCRIPTION
**Pull request by sabetech**
In this pull request:
- Tested the Calculate and Operate functions in the calculator
- Testing the components in the pages folder with snapshots

### CODE REVIEW

File: src/tests/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import operate from '../logic/operate';
+
+test('Test operate Logic (Addition) | operate(6, 9, +)', () => {
+  expect(operate(6, 9, '+')).toEqual('15');
+});
```
Comment (DeliceLydia) [inline src/tests/operate.test.js]: - [ ] I suggest that you increase tests here you tested the `+` operator only please test all operators you think a calculator can perform

File: src/tests/calculator.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+test('Test Calculate Logic Calculate({ total: 3, next: 6, operation: x }, =)', () => {
+  const obj = { total: '3', next: '6', operation: 'x' };
+  expect(calculate(obj, '=')).toEqual({ total: '18', next: null, operation: null });
+});
+
+test('Test Calculate Logic Calculate({ total: 5, next: 6, operation: - }, =)', () => {
+  const obj = { total: '5', next: '6', operation: '-' };
+  expect(calculate(obj, '=')).toEqual({ total: '-1', next: null, operation: null });
+});
```
Comment (DeliceLydia) [inline src/tests/calculator.test.js]: - [ ] Same as here please add tests for this `calculate` module test each scenario and functionalities

File: src/tests/Home.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Home from '../pages/Home';
+
+describe('Calculator pages that should render correctly', () => {
+  it('Home page should render correctly', () => {
+    const tree = renderer.create(<Home />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (DeliceLydia) [inline src/tests/Home.test.js:10]: - [ ] I can see that you tested this component but consider testing all components like `Quote`, `navbar`, `display`.. and remember using jest snapshots to them

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Kudos on:
- Awesome README
- Following git workflow
- Setting up well linters


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/tests/Home.test.js (hunk 3 above)
Comment (mateo951) [reply src/tests/Home.test.js:10]: Addressed ✔

File: src/tests/calculator.test.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  test('Test Calculate Logic Calculate({ total: 7, next: 2, operation: x }, =)', () => {
+    const obj = { total: '7', next: '2', operation: 'x' };
+    expect(calculate(obj, '=')).toEqual({ total: '14', next: null, operation: null });
+  });
+
+  test('Test Calculate Logic Calculate({ total: 10, next: 2, operation: ÷ }, =)', () => {
+    const obj = { total: '10', next: '2', operation: '÷' };
+    expect(calculate(obj, '=')).toEqual({ total: '5', next: null, operation: null });
+  });
+});
```
Comment (mateo951) [inline src/tests/calculator.test.js:23]: Addressed 📝

File: src/tests/operate.test.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  });
+
+  test('Test operate Logic (Divide) | operate(10, 5, ÷)', () => {
+    expect(operate(10, 5, '÷')).toEqual('2');
+  });
+
+  test('Test operate Logic (Modulo) | operate(9, 5, %)', () => {
+    expect(operate(9, 5, '%')).toEqual('4');
+  });
+});
```
Comment (mateo951) [inline src/tests/operate.test.js:23]: Addressed 😎

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 📝 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**